frontend/src/pages/Registration.js

```diff
@@ -0,0 +1,222 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { Formik, Form, Field } from "formik";
+import { useTranslation } from "react-i18next";
+import Button from "../components/Button";
+import InputField from "../components/InputField";
+import { signUp } from "../communications/userApi";
+import "../communications/mockForUserApi";
+import PopUpModal from "../components/PopUpModal";
+import schema from "../communications/Validation";
+
+export default function Registration() {
+  const { t } = useTranslation();
+  const [isSignUpAccepted, setIsSignUpAccepted] = useState();
+  const [modalShow, setModalShow] = React.useState(false);
+
+  const handleSubmit = async (value) => {
+    try {
+      console.log(value);
+      const response = await signUp(value);
+      console.log(response.status);
+      setIsSignUpAccepted(response.status);
+      console.log(isSignUpAccepted);
+    } catch (error) {
+      console.log(error);
+      console.log("error in handlesubmit");
+      setIsSignUpAccepted(error.status);
+    } finally {
+      setModalShow(true);
+    }
+  };
+
+  let history = useHistory();
+  const redirect = () => {
+    history.push("/login");
+  };
+
+  return (
+    <div className="d-flex flex-column justify-content-center align-content-center mx-auto col-10 col-md-4 min-vh-100">
+      <Formik
+        initialValues={{
+          name: "",
+          email: "",
+          password: "",
+          passwordConfirmation: "",
+          address: "",
+          taxNumber: "",
+          acceptedTerms: false,
+          entity: "privatePerson",
+        }}
+        onSubmit={handleSubmit}
+        validationSchema={schema(
+          t("registration.nameRequired"),
+          t("registration.invalidemail"),
+          t("Email is required"),
+          t("required"),
+          t("passwordFailMessage"),
+          t("mustAcceptTandT"),
+          t("registration.passwordMatch"),
+          t("required")
+        )}
+      >
+        {({ errors, touched }) => (
+          <Form>
+            <h1 className="text-center">{t("registration.title")}</h1>
+            <div>
+              <div className="my-3">
+                <InputField
+                  label={t("registration.fullName")}
+                  name="name"
+                  placeholder={t("registration.fullName")}
```

Comment:
Van, hogy a placeholder és a label nem egyezik? Ha nincs, akkor elég lenne csak mondjuk a labelt átadni és a komponensben belül a placeholder-t is a label szerint beállítani.

Reply:
kijavítva

Reply:
a label és a placeholder még mindig ugyan az és át van adva mindkettő, szerintem ezt akarta elkerülni Roland is, hasonlóan lejjebb a többi inputfieldnél is 🤔